### ISSUE TITLE
Should I move GIMP-Scripts into here?

### ISSUE THREAD

**Issue by patdavid** [pixlsus/Scripts#1]
Should I just move the GIMP scripts into this repo (perhaps as a sub-folder?)?

**Comment by paperdigits** [comment]
I think that's a good idea... Maybe just one repo for all user contributed scripts, presets, etc etc, else we'll end up with a repo for every program.

**Comment by paperdigits** [comment]
Should we move all the files in here? RT & DT as well?

**Comment by patdavid** [comment]
@paperdigits - seems like it would make sense, right?

**Comment by paperdigits** [comment]
Yeah, I mean, I think one big repo is nicer than 5 repos  that barely have anything... for things like this, I've taken to just symlinking the data where it needs to go.
